modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/CacheLateAffinityAssignmentTest.java

```diff
@@ -2134,7 +2134,7 @@ public void testNoForceKeysRequests() throws Exception {
 
         final AtomicInteger nodeIdx = new AtomicInteger();
 
-        final long stopTime = System.currentTimeMillis() + 60_000;
+        final long stopTime = System.currentTimeMillis() + GridTestUtils.SF.applyUB(400_000, 60_000);
```

Comment:
It seems test stop time is raised here to 400 seconds. Should we reduce it?

Reply:
No, this means that max time 60 seconds and min 400_000 * 0.1 miles = 40 seconds.

Reply:
fixed